package.json

```diff
@@ -2741,5 +2741,11 @@
 	"optionalDependencies": {
 		"bufferutil": "^4.0.6",
 		"utf-8-validate": "^5.0.9"
+	},
+	"__metadata": {
```

Comment:
Do you know where this came from? I haven't seen it before (although I'm on an older version of vsce because I had issues with a previous upgrade, so perhaps a newer version is doing this?)